parallel_build.csh

```diff
@@ -30,7 +30,7 @@ while ($#argv)
    shift
 end
 
-if (! -d ${ESMADIR}/@env) then
+if (! -d ${ESMADIR}/@env || ! -d ${ESMADIR}/env@ || ! -d ${ESMADIR}/env) then
```

Comment:
My PR will include a macro which can change this to:
```cmake
esma_mepo_style(env mepo_env)
```
And by this point, the macro is available.   One then uses `${mepo_env}` below to eliminate the `if` chain.

Reply:
I don't think your macro will work here. This is a C-shell script and there is not really a prettier way to fix this.

Reply:
Well, I mean, I could make a pseudo-csh function but...eh. I'd rather rewrite in Bash at that point!